### PR TITLE
translation, password

### PR DESCRIPTION
**Pull request by jennyhelbing**

### CODE REVIEW

File: src/Enhavo/Bundle/AppBundle/Resources/translations/EnhavoAppBundle.en.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  delete: 'Delete'
+  preview: 'Preview'
+  create: 'Create'
+  yes: 'Yes'
+  no: 'No'
+  title: 'Title'
+  upload: 'Upload data'
+form:
+  delete:
+    question: 'Wollen sie dieses Element wirklich löschen?'
```
Comment (gseidel) [inline src/Enhavo/Bundle/AppBundle/Resources/translations/EnhavoAppBundle.en.yml]: English: "Would you like to delete this element?"

File: src/Enhavo/Bundle/ArticleBundle/Resources/translations/EnhavoArticleBundle.en.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  label:
+    article: 'Article'
+    content: 'Content'
+    seo: 'SEO'
+    preview: 'Preview'
+    id: 'ID'
+    title: 'Title'
+label:
+  article: 'Article'
+  yes: 'Yes'
```
Comment (gseidel) [inline src/Enhavo/Bundle/ArticleBundle/Resources/translations/EnhavoArticleBundle.en.yml]: use it in EnhavoAppBundle instead

File: src/Enhavo/Bundle/ArticleBundle/Resources/translations/EnhavoArticleBundle.en.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      publication_date: 'Publication date'
+  label:
+    article: 'Article'
+    content: 'Content'
+    seo: 'SEO'
+    preview: 'Preview'
+    id: 'ID'
+    title: 'Title'
+label:
+  article: 'Article'
```
Comment (gseidel) [inline src/Enhavo/Bundle/ArticleBundle/Resources/translations/EnhavoArticleBundle.en.yml]: we can use the article.label.article instead

File: src/Enhavo/Bundle/UserBundle/Controller/ChangePasswordController.php [hunk 4]
```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the FOSUserBundle package.
```
Comment (gseidel) [inline src/Enhavo/Bundle/UserBundle/Controller/ChangePasswordController.php]: Remove this comments

File: src/Enhavo/Bundle/UserBundle/Controller/ChangePasswordController.php [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+use Symfony\Component\BrowserKit\Response;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+
+/**
+ * Controller managing the password change
+ *
+ * @author Thibault Duplessis <thibault.duplessis@gmail.com>
```
Comment (gseidel) [inline src/Enhavo/Bundle/UserBundle/Controller/ChangePasswordController.php]: Remove author

File: src/Enhavo/Bundle/PageBundle/Resources/config/doctrine/Tags.orm.yml [hunk 6]
```diff
@@ -0,0 +1,31 @@
+Enhavo\Bundle\PageBundle\Entity\Tags:
```
Comment (gseidel) [inline src/Enhavo/Bundle/PageBundle/Resources/config/doctrine/Tags.orm.yml]: This is not part of this pull request, please delete this file

File: src/Enhavo/Bundle/PageBundle/Form/Type/TagType.php [hunk 7]
```diff
@@ -0,0 +1,33 @@
+<?php
```
Comment (gseidel) [inline src/Enhavo/Bundle/PageBundle/Form/Type/TagType.php]: This is not part of this pull request, please delete this file

File: src/Enhavo/Bundle/PageBundle/Entity/Tags.php [hunk 8]
```diff
@@ -0,0 +1,140 @@
+<?php
```
Comment (gseidel) [inline src/Enhavo/Bundle/PageBundle/Entity/Tags.php]: This is not part of this pull request, please delete this file

File: src/Enhavo/Bundle/PageBundle/Controller/TagsController.php [hunk 9]
```diff
@@ -0,0 +1,10 @@
+<?php
```
Comment (gseidel) [inline src/Enhavo/Bundle/PageBundle/Controller/TagsController.php]: This is not part of this pull request, please delete this file

File: src/Enhavo/Bundle/ArticleBundle/Resources/translations/EnhavoArticleBundle.en.yml (hunk 2 above)
Comment (jennyhelbing) [inline src/Enhavo/Bundle/ArticleBundle/Resources/translations/EnhavoArticleBundle.en.yml]: what do you mean ?
